STRATEGIC-INITIATIVES.md

```diff
@@ -0,0 +1,46 @@
+# Strategic Initiatives
+
+At any one time the Node.js project has a number of strategic initiatives
+underway.  With the less-tangible approach to the work that the Community 
+Committee does, having a way to track this work is important.
+
+For each strategic initiative, the Community Committee's goal is to have a
+single point of reference - a champion - that understands what's happening
+with the initiative to the fullest extent posssible, and can report on the
+initiative as needed.
+
+A review of the initiatives will be a standing item on the Community Committee agenda (even
+if the update is 'nothing new') as a way to ensure they are active
+and have the support needed.
+
+# Current Initiatives
+
+| Initiative         | Champion                        | Links                                                            |
+|--------------------|---------------------------------|------------------------------------------------------------------|
+| NodeTogether       | Rachel White                    | https://github.com/nodejs/community-committee/issues/63          |
+| User Feedback      | Michael Dawson                  | https://github.com/nodejs/user-feedback                          |
```

Comment:
I think we have differing definitions of what we mean by Champions here. I asked @mhdawson to help in the formation of the User Feedback team since the members volunteering to kick off the initiative were all new to the CommComm and were and are at most Observers. We referred to that "role" as our Champion to the CommComm so we knew that we would have someone committed to mentoring this nascent team to success.

Since the following line indicates @PatrickHeneise as the Champion for Community Events, who is not a CommComm member, the "Champion" for User Feedback should be Dan Shaw (me).

Reply:
@dshaw for context, Community Events was an early effort in the CommComm that was (to the best of my understanding) driven by @PatrickHeneise before @rachelnicole and myself were Co-chairs and before we started working on driving members of the CommComm as Champions.

I added @PatrickHeneise as a champion in this document since he _was_ (again, to the best of my knowledge) the champion. If anything, I can remove @PatrickHeneise. As @williamkapke pointed out in the Public Board meeting which I attended last month, that effort has not seen any follow-up from members or observers, largely because it seems the people that were driving it have dipped out.

We likely need to assess its relevance and place in the CommComm as soon as possible. I've been working to put the pieces in place for this kind of assessment - this document being one of them.